client/extensions/woocommerce/state/wc-api/README.md

```diff
@@ -0,0 +1,90 @@
+wc-api
+======
+
+**The WooCommerce API access module for Calypso.**
+
+This module connects the application to a WooCommerce Site via its API and manages local state.
+
+## Features
+
+* Provides a Higher Order Component that allows a component to define its data needs instead of performing fetches each time.
+* Providing selectors to access data in various formats.
```

Comment:
It provides the selectors, or it provides state that we can use selectors on?

Reply:
I'd really like wc-api to be a one stop shop for its own data here, mostly so it can own its own redux state completely. Also, if we curry the selectors here with the siteId, that's one less thing to worry about when getting data from state.